components/calendar/src/types.rs

```diff
@@ -180,7 +180,7 @@ impl FormattingEra {
 }
 
 /// Year information for a year that is specified with an era
-#[derive(Copy, Clone, Debug, PartialEq)]
+#[derive(Copy, Clone, Debug, PartialEq, Default)]
```

Comment:
Why did you impl Default? There's not really a default year.

Reply:
Makes it convenient to construct. Generally I feel like non-exhaustive types should be constructible. We're not rigidly following that guideline here, but I'm trying to follow it where possible.

I don't mind removing this for now.

Reply:
Yeah we might consider adding a constructor, but I think avoid the scope creep for now.